source/common/router/config_impl.cc

```diff
@@ -376,9 +376,20 @@ CorsPolicyImpl::CorsPolicyImpl(const envoy::config::route::v3::CorsPolicy& confi
   }
 }
 
-ShadowPolicyImpl::ShadowPolicyImpl(const RequestMirrorPolicy& config) {
-  cluster_ = config.cluster();
+const envoy::config::route::v3::RouteAction::RequestMirrorPolicy&
+validateClusterSpecifier(const envoy::config::route::v3::RouteAction::RequestMirrorPolicy& config) {
```

Comment:
The input and output of this method is the same and not changed.
If it's just a validation function then I see 2 ways:
1. Change it to return void and throw exception.
2. Change it to return StatusOr, and throw exception by its caller (`RouteEntryImplBase` c'tor).

Reply:
Done.